MANIFEST.in

```diff
@@ -1,10 +1,3 @@
-graft test
```

Comment:
any reason these are now removed?

Reply:
Yes, these files were not needed for building and installing the package. They were left in from the original packaging implementation. This just cleans that up.